reactnative/README.md

```diff
@@ -0,0 +1,66 @@
+# React Native Input Mask
+
+### This is beta and is a still a WIP!
+
+## Getting started
+
+First, install it.
+
+```bash
+npm i react-native-text-mask --save
+```
+
+Then, require it and use it.
+
+```js
+import React, { Component } from 'react'
+import { View } from 'react-native'
+import MaskedInput from 'react-native-text-mask'
+
+class MyComponent extends Component {
+  render() {
+    return (
+      <View>
+        <MaskedInput mask={['(', /[1-9]/, /\d/, /\d/, ')', ' ', /\d/, /\d/, /\d/, '-', /\d/, /\d/, /\d/, /\d/]} />
+      </View>
+    )
+  }
+}
+```
+
+`<MaskedInput/>` is fully compatible with the `<TextInput/>` element. So, you can
+pass any style, a placeholder attribute, and all other `TextInput` props.
+
+For example, the following works:
+
+```js
+<MaskedInput
+  mask={['(', /[1-9]/, /\d/, /\d/, ')', ' ', /\d/, /\d/, /\d/, '-', /\d/, /\d/, /\d/, /\d/]}
+  placeholder="Enter a phone number"
+  guide={false}
```

Comment:
Why do you show `guide={false}` here? I think it's better to show the minimum required API in the example.